names/names.py

```diff
@@ -11,10 +11,11 @@
 f.close()
 
 duplicates = []
-for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+names_1_set = set(names_1)
```

Comment:
Hahhaha, you went straight to the Stretch problem! Great work; remember to add in a binary search tree implementation when you get a shot. Anyway, excellent work with the `intersection` method on sets; that's really nifty!